### PR TITLE
feat(w1r3): initial Python benchmark

### PR DESCRIPTION
**Pull request by coryan**

### CODE REVIEW

File: w1r3/python/w1r3.py [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+    )
+    cpu = meter.create_histogram(
+        _CPU_HISTOGRAM_NAME,
+        unit=_CPU_HISTOGRAM_UNIT,
+        description=_CPU_HISTOGRAM_DESCRIPTION,
+    )
+
+    for i in range(args.iterations):
+        (transport_name, client) = prng.choice(clients)
+        object_size = prng.choice(args.object_size)
```
Comment (cojenco) [inline w1r3/python/w1r3.py:101]: Just wanted to confirm if my understanding is right… so we’re not randomly choosing from a range, but instead from a list of object sizes, e.g  100KB, 2MiB, 100MB

File: w1r3/python/w1r3.py [hunk 2]
```diff
@@ -0,0 +1,377 @@
...
+
+    # Instrument Python's `requests` library to generate spans.
+    RequestsInstrumentor().instrument()
+
+    clients = make_clients(args)
+    prng = random.Random()
+    prng.seed()
+    random_data = bytes([prng.randint(0, 255) for _ in range(0, max(args.object_size))])
+
+    worker(clients, instance, prng, random_data, tracer, meter, args)
```
Comment (cojenco) [inline w1r3/python/w1r3.py:76]: It seems like we’re running a worker in a single process. Will this be ran concurrently?

**Review by cojenco — APPROVED** [review]

File: w1r3/python/w1r3.py (hunk 2 above)
Comment (coryan) [reply w1r3/python/w1r3.py:76]: Yes, we end up running N processes on the same VM.  We use a single thread per-process to make CPU (and eventually memory) measurements easier.

File: w1r3/python/w1r3.py (hunk 1 above)
Comment (coryan) [reply w1r3/python/w1r3.py:101]: Yes. We want to upload the object, a range won't be as useful.  We have lots of data showing nothing changes much if you do range reads of size X vs. reading an object of size X.

**Pull request merged**